pkg/cmd/cli/cmd/newapp.go

```diff
@@ -99,6 +101,16 @@ func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out io.Writer)
 				glog.Fatalf("Error: %v", err)
 			}
 
+			label := cmdutil.GetFlagString(c, "label")
+			if len(label) == 0 {
+				label = util.GetLabel()
```

Comment:
> Ah, wording. The flag should be "--labels"

I am going to change it to "--selector" as [kubectl delete](https://github.com/GoogleCloudPlatform/kubernetes/blob/860257e01f5fb5cc99427a4fbd970b70527de814/pkg/kubectl/cmd/delete.go#L71) uses it.